src/jaxsim/api/contact.py

```diff
@@ -357,12 +365,10 @@ def jacobian(
 
     # Adjust the output representation.
     match output_vel_repr:
-
```

Comment:
Sure! A possible idea to avoid this kind of formatting conflicts might be to use `ruff` formatter instead of `black`, which automatically removed these kind of blank lines.

Reply:
Wait, I want to keep these lines, not removing them :)

Reply:
Black removes these lines only in a particular -and kind of unfortunate- version, I think either the 2022 or 2023 one. Now I pinned 2024 [project-wise](https://github.com/ami-iit/jaxsim/blob/35f25b1d006a604257684bc9b89c622d8400ccec/setup.cfg#L72).

Reply:
Done in